src/libs/ascent/runtimes/flow_filters/ascent_runtime_rendering_filters.cpp

```diff
@@ -1218,10 +1230,77 @@ DefaultRender::execute()
 	      scene_bounds.Z.Max = d_bounds[5];
 	  }
 
-          vtkh::Render render = detail::parse_render(render_node,
-                                                     scene_bounds,
-                                                     image_name);
-          renders->push_back(render);
+	  if(is_auto_camera)
+	  { 
+            DataObject *source
+              = graph().workspace().registry().fetch<DataObject>("source_object");
+            
```

Comment:
Auto-cam selection is being run against the original data.

This will work well for many cases, longer term we should think about if we need to push auto cam params into the renderer, to get the actual data.